integration/tests/posit/connect/test_content.py

```diff
@@ -23,7 +23,17 @@ def test_count(self):
         assert self.client.content.count() == 1
 
     def test_get(self):
-        assert self.client.content.get(self.content["guid"]) == self.content
+        item = self.client.content.get(self.content["guid"])
+        # get() always includes owner, tags, and vanity_url. Owner data is always present in all
+        # content, tags and vanity_url is only present if explicitly set in the content.
+        # Check that essential fields match instead of exact equality
+        for key in self.content:
+            assert key in item
+            assert item[key] == self.content[key]
+        # Also verify we have the additional fields that should always be included
+        assert "owner" in item
+        assert "tags" not in item
+        assert "vanity_url" not in item
```

Comment:
Yeah, looks like this was introduced in 2024.06.0. You can key these tests off of https://github.com/posit-dev/posit-sdk-py/blob/main/integration/tests/posit/connect/__init__.py#L8